config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
I will take a look at this for sure. thanks for bringing it up. I think the contents of `log4j2.properties` are up for debate and iteration anyways, so I'd like to leave concerns inside this file to a followup PR.